scripts/globals/spells/trust/monberaux.lua

```diff
@@ -14,16 +14,30 @@ spellObject.onSpellCast = function(caster, target, spell)
 end
 
 spellObject.onMobSpawn = function(mob)
-    -- TODO: Summon: I am Doctor Monberaux. My services are available for any affliction.
     -- TODO: Summon (1 dose of Final Elixir ready): I received a vial of a most valuable medicine. It should prove useful in times of emergency.
     -- TODO: Summon (2 doses of Final Elixir ready): I received two vials of a most valuable medicine. They should prove useful in times of emergency.
     xi.trust.message(mob, xi.trust.message_offset.SPAWN)
 
-    -- local healingMoveCooldown = 10
+    local healingMoveCooldown = 3
 
     -- Tends to be particular about which potions to use. Seems to favor healing for just the
     -- right amount of HP instead of defaulting to the highest-rank potion.
-    -- mob:addSimpleGambit(ai.t.PARTY, ai.c.HP_MISSING, 500, ai.r.MS, ai.s.SPECIFIC, 4236, healingMoveCooldown) -- Max Potion
+    mob:addMod(xi.mod.MPP, -80)
+    mob:addMod(xi.mod.HPP, 50)
```

Comment:
Where are these values from?

Reply:
The -Max MP is from BGWiki, which says he has -80% MP. The wiki also says his stats and health scale as if he was a tank job, and yet he had very low health from my testing on my own server (~1800 for paladin elvaan). I couldn't figure out a way to edit his stats normally so I gave him an HP modifier. I'll get rid of it if it's not acceptable, as I wasn't sure how to make him not so squishy.

Reply:
fun fact: he basically IS an PLD elvaan, even has the traits.

Our stat basis **should** be coming from the rankings in mob family system but right now I don't know if that is really the case (I recall it being bs made up in core instead at one point) **and** I think all or almost all trusts are using the same generic humanoid for their family ID (145 iirc) when we probably need several (especially for the non humanoid ones like the Goobue).

Reply:
I would consider the mp one legit, for whatever reason SE decided to nerf his native pld mp pool even tho he has no spells.